src/bms/parse/model.rs

```diff
@@ -200,12 +201,12 @@ pub struct Notes {
     pub wav_files: HashMap<ObjId, PathBuf>,
     // objects stored in obj is sorted, so it can be searched by bisection method
     /// BGM objects, indexed by time. #XXX01:ZZ... (BGM placement)
-    pub bgms: BTreeMap<ObjTime, Vec<ObjId>>,
-    /// All note objects, indexed by ObjId. #XXXYY:ZZ... (note placement)
-    pub objs: HashMap<ObjId, Vec<Obj>>,
+    pub bgms: MultiMap<ObjTime, ObjId>,
```

Comment:
`MultiMap<K, V>` uses `HashMap<K, Vec<V>>` internally, so I think it will get performance worse on scanning bgm notes chronologically.